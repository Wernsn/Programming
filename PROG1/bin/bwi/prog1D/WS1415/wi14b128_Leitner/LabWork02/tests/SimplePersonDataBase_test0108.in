2
c
y
2015
1
1
2
b
z
2014
1
1
2
a
x
2016
1
1
1
5
x
5
y
5
z
5
a
2
cfirst
ylast
2015
1
1
2
bfirst
lasta
2014
1
1
2
afirst
lastb
2016
1
1
1
5
x
5
a
5
y
5
last
5
lasta
0


