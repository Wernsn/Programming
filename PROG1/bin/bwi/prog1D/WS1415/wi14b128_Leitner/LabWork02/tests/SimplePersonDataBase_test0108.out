Welcome to the SimplePersonDatabase.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: 
ID  Lastname             Firstname            Birth
--------------------------------------------------------------------
001 y                    c                    Thursday, 01.01.2015
002 z                    b                    Wednesday, 01.01.2014
003 x                    a                    Friday, 01.01.2016
--------------------------------------------------------------------
3 Person(s) in database, 17 free.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
***
003 x                    a                    Friday, 01.01.2016
***
1 match(es) found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
***
001 y                    c                    Thursday, 01.01.2015
***
1 match(es) found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
***
002 z                    b                    Wednesday, 01.01.2014
***
1 match(es) found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
no matches found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: 
ID  Lastname             Firstname            Birth
--------------------------------------------------------------------
001 y                    c                    Thursday, 01.01.2015
002 z                    b                    Wednesday, 01.01.2014
003 x                    a                    Friday, 01.01.2016
004 ylast                cfirst               Thursday, 01.01.2015
005 lasta                bfirst               Wednesday, 01.01.2014
006 lastb                afirst               Friday, 01.01.2016
--------------------------------------------------------------------
6 Person(s) in database, 14 free.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
***
003 x                    a                    Friday, 01.01.2016
***
1 match(es) found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
no matches found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
***
001 y                    c                    Thursday, 01.01.2015
004 ylast                cfirst               Thursday, 01.01.2015
***
2 match(es) found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
***
005 lasta                bfirst               Wednesday, 01.01.2014
006 lastb                afirst               Friday, 01.01.2016
***
2 match(es) found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: enter lastname: 
***
005 lasta                bfirst               Wednesday, 01.01.2014
***
1 match(es) found.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: 
Thank you for using the SimplePersonDatabase.

